libs/modelviewer/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "@cc-components/modelviewer",
+  "version": "0.0.1",
+  "type": "module",
+  "main": "./dist/src/index.js",
+  "module": "./dist/src/index.js",
+  "types": "./dist/src/index.d.ts",
+  "scripts": {
+    "build": "tsc -b -f",
+    "test": "vitest",
+    "test:silent": "vitest --silent --run",
+    "test:coverage": "vitest run --coverage"
+  },
+  "dependencies": {
+    "@equinor/echo-3d-viewer": "^0.0.1",
+    "@cognite/reveal": "4.2.2",
+    "@cognite/sdk": "8.2.0",
+    "three": "0.154.0",
+    "rxjs": "^7.5.7"
+  },
+  "devDependencies": {
+    "@equinor/fusion-framework-module": "^4.2.5",
```

Comment:
Ìf possible I would prefer the fusion-framework dependencies to be declared in the root package.json to ensure consistency